dali/test/python/test_operator_slice.py

```diff
@@ -338,70 +358,89 @@ def iter_setup(self):
+                        ((200, 400, 3), "HWC", (0, 1), None, types.FLOAT16, types.FLOAT16),
+                        ((200, 400, 3), "HWC", (2,), None, types.FLOAT, None),
+                        ((200,), "H", (0,), None, types.FLOAT, None),
+                        ((200,), "H", None, "H", types.FLOAT, None),
+                        ((200, 400), "HW", (1,), None, types.FLOAT, None),
+                        ((200, 400), "HW", None, "W", types.FLOAT, None),
+                        ((80, 30, 20, 3), "DHWC", (2, 1, 0), None, types.FLOAT, None),
+                        ((80, 30, 20, 3), "DHWC", (0, 1, 2), None, types.FLOAT, None),
+                        ((80, 30, 20, 3), "DHWC", (2, 1), None, types.FLOAT, None),
+                        ((80, 30, 20, 3), "DHWC", None, "WHD", types.FLOAT, None),
+                        ((80, 30, 20, 3), "DHWC", None, "DHW", types.FLOAT, None),
+                        ((80, 30, 20, 3), "DHWC", None, "WH", types.FLOAT, None),
+                        ((80, 30, 20, 3), "DHWC", None, "C", types.FLOAT, None)
+                    ]:
                 normalized_anchor = np.random.choice([True, False])
                 normalized_shape = np.random.choice([True, False])
-                yield check_slice_synth_data_vs_numpy, device, batch_size, \
-                    input_shape, layout, axes, axis_names, normalized_anchor, normalized_shape, \
-                    input_type, output_type
+                yield (check_slice_synth_data_vs_numpy, device, batch_size,
+                       input_shape, layout,
+                       axes, axis_names,
+                       normalized_anchor, normalized_shape,
+                       input_type, output_type)
 
 
 def check_slice_vs_fused_decoder(device, batch_size, axes, axis_names):
     eii_args = [SliceArgsIterator(batch_size, image_layout="HWC", axes=axes, axis_names=axis_names)
                 for k in range(2)]
     compare_pipelines(
-        SlicePipeline(device, batch_size, iter(eii_args[0]), axes=axes, axis_names=axis_names, is_fused_decoder=False),
-        SlicePipeline(device, batch_size, iter(eii_args[1]), axes=axes, axis_names=axis_names, is_fused_decoder=True),
+        SlicePipeline(device, batch_size, iter(eii_args[0]),
+                      axes=axes, axis_names=axis_names, is_fused_decoder=False),
+        SlicePipeline(device, batch_size, iter(eii_args[1]),
+                      axes=axes, axis_names=axis_names, is_fused_decoder=True),
         batch_size=batch_size, N_iterations=3)
 
 
 def test_slice_vs_fused_decoder():
     for device in ["cpu", "gpu"]:
         for batch_size in {1}:
-            for axes, axis_names in \
-                [(None, "WH"), (None, "HW"),
-                ((1,0), None), ((0,1), None)]:
+            for axes, axis_names in [
+                        (None, "WH"), (None, "HW"),
+                        ((1, 0), None), ((0, 1), None)
+                    ]:
```

Comment:
```suggestion
                        ((1, 0), None), ((0, 1), None)]:
```
Would be preferred, I think

Reply:
Again - not when there's a new line after the opening brace.